text/0005-identity.md

```diff
@@ -0,0 +1,182 @@
+Proofs come in all sorts of formats. eg:
+    > photocopies of passports, birth certificates, and utility bills. 
+    For companies it’s a bundle of incorporation and ownership structure 
+    documents.
+- **Attestation**: When a third party validates that according to their records, 
+the claims are true. 
+    > For example a University may attest to the fact that someone studied there 
+    and earned a degree. An attestation from the right authority is more robust 
+    than a proof, which may be forged. However, attestations are a burden on the 
+    authority as the information can be sensitive. This means that the 
+    information needs to be maintained so that only specific people can access 
+    it.
+
+### Issuer, Verifier
+- **Issuer**: an entity that generates claims
+- **Verifier**: an entity that confirms the validity of a claim
+
+### Private & Public Signing Keys (Keypair Cryptography)
+- **private key** (signing key) is used to sign documents, and is kept secret by
+the issuer.
+- **public key** (verification key) used to verify the signature and ensure the 
+document has not been tampered with, and it does not need to be kept secret.
+
+### Verifiable Claim
+- **Verifiable claims** are a standard way of defining, exchanging, and 
+verifying digital credentials. The strength of the claim depends on the degree 
+of trust the verifier has in the issuer. For example, if a bank issues a claim 
+saying that you have a certain credit card number, a merchant can rely on the 
+claim if the merchant has a high degree of trust in the bank.
+
+### Revocation
+* "I take back what I just said"
+
+## Qri Profiles
+TODO
+
+- Identity on qri is connected to a cryptographic keypair
+- Profile connects descriptive info to a keypair
+- Profile info is _opt-in_. Requred values are randomly generated
+- Profile keypairs are used to sign datasets
+- Profile keys are _not_ the same as IPFS keys
+- Peernames are _not_ enforced as unique at this level
+- Qri keypairs sign dataset body hashes as a verifiable claim
+
+
+### Identity & Registries
+TODO
+
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+TODO - finish
+
+Profile
+```golang
+// Profile is a definition of a profile on qri
+type Profile struct {
+  ID       string `json:"id"`
+  PrivKey  string `json:"privkey,omitempty"`
+  Peername string `json:"peername"`
+  // Created timestamp
+  Created time.Time `json:"created"`
+  // Updated timestamp
+  Updated time.Time `json:"updated"`
+  // specifies weather this is a user or an organization
+  Type string `json:"type"`
+  // user's email address
+  Email string `json:"email"`
+  // user name field. could be first[space]last, but not strictly enforced
+  Name string `json:"name"`
+  // user-filled description of self
+  Description string `json:"description"`
+  // url this user wants the world to click
+  HomeURL string `json:"homeurl"`
+  // color this user likes to use as their theme color
```

Comment:
Is this html-like "#rrggbb"? Are certain web color names allowed?

Reply:
I think we should go with #rrggbb or empty string (which should represent the default)